[2019-02-22 13:10:21.555] [INFO] console - Express server listening on :8093, in development mode
[2019-02-22 13:10:21.565] [INFO] console - mongo连接成功
[2019-02-22 13:10:21.567] [INFO] console - mongo连接打开
[2019-02-22 13:12:20.456] [INFO] console - ******** getResFn error ******** 
[2019-02-22 13:12:20.458] [INFO] console - ReferenceError: ctx is not defined
    at /Users/roger/ML/auth-server/controllers/auth/app.js:85:9
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:47:12)
[2019-02-22 13:12:20.466] [ERROR] console - ReferenceError: ctx is not defined
    at /Users/roger/ML/auth-server/controllers/auth/app.js:85:9
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)
    at router (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:47:12) 'ReferenceError: ctx is not defined\n    at /Users/roger/ML/auth-server/controllers/auth/app.js:85:9\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)\n    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)\n    at router (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:47:12)'
[2019-02-22 13:12:20.470] [ERROR] console - (node:74624) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated
[2019-02-22 13:13:03.871] [INFO] console - Express server listening on :8093, in development mode
[2019-02-22 13:13:03.880] [INFO] console - mongo连接成功
[2019-02-22 13:13:03.882] [INFO] console - mongo连接打开
[2019-02-22 13:13:18.392] [INFO] console - Express server listening on :8093, in development mode
[2019-02-22 13:13:18.401] [INFO] console - mongo连接成功
[2019-02-22 13:13:18.403] [INFO] console - mongo连接打开
[2019-02-22 13:13:19.952] [INFO] console - ******** getResFn error ******** 
[2019-02-22 13:13:19.954] [INFO] console - { 401: need authorization
    at fail (/Users/roger/ML/auth-server/utils/controller/index.js:47:19)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:91:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 401 }
[2019-02-22 13:13:19.962] [ERROR] console - { 401: need authorization
    at fail (/Users/roger/ML/auth-server/utils/controller/index.js:47:19)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:91:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 401 } '401: need authorization\n    at fail (/Users/roger/ML/auth-server/utils/controller/index.js:47:19)\n    at /Users/roger/ML/auth-server/controllers/auth/app.js:91:13\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)\n    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)'
[2019-02-22 13:13:19.965] [ERROR] console - (node:74679) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated
[2019-02-22 13:13:54.018] [INFO] console - Express server listening on :8093, in development mode
[2019-02-22 13:13:54.026] [INFO] console - mongo连接成功
[2019-02-22 13:13:54.028] [INFO] console - mongo连接打开
[2019-02-22 13:13:55.976] [INFO] console - ******** getResFn error ******** 
[2019-02-22 13:13:55.978] [INFO] console - { JsonWebTokenError: jwt malformed
    at Object.module.exports [as verify] (/Users/roger/ML/auth-server/node_modules/jsonwebtoken/verify.js:53:17)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:88:39
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 'JsonWebTokenError', message: 'jwt malformed' }
[2019-02-22 13:13:55.986] [ERROR] console - { JsonWebTokenError: jwt malformed
    at Object.module.exports [as verify] (/Users/roger/ML/auth-server/node_modules/jsonwebtoken/verify.js:53:17)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:88:39
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 'JsonWebTokenError', message: 'jwt malformed' } 'JsonWebTokenError: jwt malformed\n    at Object.module.exports [as verify] (/Users/roger/ML/auth-server/node_modules/jsonwebtoken/verify.js:53:17)\n    at /Users/roger/ML/auth-server/controllers/auth/app.js:88:39\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)\n    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)'
[2019-02-22 13:13:55.989] [ERROR] console - (node:74703) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated
[2019-02-22 13:14:13.684] [ERROR] console - (node:74703) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
[2019-02-22 13:14:24.798] [INFO] console - ******** getResFn error ******** 
[2019-02-22 13:14:24.800] [INFO] console - { JsonWebTokenError: invalid token
    at Object.module.exports [as verify] (/Users/roger/ML/auth-server/node_modules/jsonwebtoken/verify.js:65:17)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:88:39
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 'JsonWebTokenError', message: 'invalid token' }
[2019-02-22 13:14:24.803] [ERROR] console - { JsonWebTokenError: invalid token
    at Object.module.exports [as verify] (/Users/roger/ML/auth-server/node_modules/jsonwebtoken/verify.js:65:17)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:88:39
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 'JsonWebTokenError', message: 'invalid token' } 'JsonWebTokenError: invalid token\n    at Object.module.exports [as verify] (/Users/roger/ML/auth-server/node_modules/jsonwebtoken/verify.js:65:17)\n    at /Users/roger/ML/auth-server/controllers/auth/app.js:88:39\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)\n    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)'
[2019-02-22 13:15:27.661] [INFO] console - Express server listening on :8093, in development mode
[2019-02-22 13:15:27.670] [INFO] console - mongo连接成功
[2019-02-22 13:15:27.672] [INFO] console - mongo连接打开
[2019-02-22 13:15:33.868] [INFO] console - ******** getResFn error ******** 
[2019-02-22 13:15:33.870] [INFO] console - { 401: authorization failed
    at fail (/Users/roger/ML/auth-server/utils/controller/index.js:47:19)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:90:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 401 }
[2019-02-22 13:15:33.876] [ERROR] console - { 401: authorization failed
    at fail (/Users/roger/ML/auth-server/utils/controller/index.js:47:19)
    at /Users/roger/ML/auth-server/controllers/auth/app.js:90:13
    at Generator.next (<anonymous>)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)
    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)
    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)
    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5
    at new Promise (<anonymous>)
    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)
    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)
    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)
    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3) name: 401 } '401: authorization failed\n    at fail (/Users/roger/ML/auth-server/utils/controller/index.js:47:19)\n    at /Users/roger/ML/auth-server/controllers/auth/app.js:90:13\n    at Generator.next (<anonymous>)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:65:19)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at toPromise (/Users/roger/ML/auth-server/node_modules/co/index.js:118:63)\n    at next (/Users/roger/ML/auth-server/node_modules/co/index.js:99:29)\n    at onFulfilled (/Users/roger/ML/auth-server/node_modules/co/index.js:69:7)\n    at /Users/roger/ML/auth-server/node_modules/co/index.js:54:5\n    at new Promise (<anonymous>)\n    at co (/Users/roger/ML/auth-server/node_modules/co/index.js:50:10)\n    at /Users/roger/ML/auth-server/utils/controller/index.js:6:9\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/Users/roger/ML/auth-server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/Users/roger/ML/auth-server/node_modules/express/lib/router/layer.js:95:5)\n    at /Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:335:12)\n    at next (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/Users/roger/ML/auth-server/node_modules/express/lib/router/index.js:174:3)'
[2019-02-22 13:15:33.879] [ERROR] console - (node:74740) [DEP0079] DeprecationWarning: Custom inspection function on Objects via .inspect() is deprecated
